zebra-network/src/constants.rs

```diff
@@ -95,6 +95,12 @@ pub const EWMA_DEFAULT_RTT: Duration = Duration::from_secs(20 + 1);
 /// better peers when we restart the sync.
 pub const EWMA_DECAY_TIME: Duration = Duration::from_secs(200);
 
+/// Linux portion of OS error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_UNIX: &str = "already in use";
+
+/// Windows portion of OS error when the port attempting to be opened is already in use.
+pub const PORT_IN_USE_WINDOWS: &str = "one usage";
```

Comment:
Rust knows the platform we're building and running on, so we can avoid having a different variable name for each platform.

```suggestion
#[cfg(unix)]
pub const PORT_IN_USE_ERROR: &str = "already in use";
#[cfg(not(unix))]
pub const PORT_IN_USE_ERROR: &str = "one usage";
```

Reply:
This simplifies a lot, thanks! I made the change at https://github.com/ZcashFoundation/zebra/pull/1535/commits/925a4fbaef5645d2e91e1de6c674e1d21d9da464